components/CurrentPetStats.jsx

```diff
@@ -1,38 +1,51 @@
 import StyledStatKey from "./Styles/StyledStatKey";
 import StyledStatsContainer from "./Styles/StyledStatsContainer";
 import StyledListItem from "./Styles/StyledListItem";
+import StyledDiv from "./Styles/StyledDiv";
+import PositionedButton from "./Styles/StyledButton";
 
```

Comment:
Also, in some cases it might possibly make sense not to outsource the styling to a separate file but to style the component directly in the component file (if the style is not reused anywhere else of course) as you have to import a lot of styles here.

Reply:
thanks, we decided to outsource all styled components to the Styled folder, to keep the components more readable